examples/lighting-app/qpg/src/LightingManager.cpp

```diff
@@ -91,6 +120,30 @@ bool LightingManager::InitiateAction(Action_t aAction, int32_t aActor, uint16_t
             new_state = kState_On;
         }
     }
+    else if (aAction == COLOR_ACTION_XY)
+    {
+        action_initiated = true;
+        if (xy.x == 0 && xy.y == 0)
+        {
+            new_state = kState_Off;
```

Comment:
is new_state used below? it seems only used on the else path when aAction is not XY pr HSV.

Reply:
It is having an indirect effect in UpdateLight, where will result in turning off the PWM.